website/public/locales/en/chat.json

```diff
@@ -3,5 +3,14 @@
   "chat_date": "{{val, datetime}}",
   "empty": "(empty)",
   "queue_info": "Your message is queued, you are at position {{ queuePosition, number, integer }} in the queue.",
-  "your_chats": "Your chats"
+  "your_chats": "Your chats",
+  "chat_config": {
+    "title": "Chat configuration",
+    "model": "Model",
+    "top_k": "top_k",
```

Comment:
I dont understand what this param is.

Reply:
This might be a bit of a long read, but It makes all of these parameters much clearer:
https://huggingface.co/blog/how-to-generate

You can also look up the documentation of huggingface